### DISCUSSION TITLE
source/PlayState.hx:298: characters 11-15 : Missing ;

### DISCUSSION THREAD

**Discussion by AshuraTheEchidna** [FunkinCrew/Funkin#1555] [Compiling help]
i had this error that i cant fix, i did everything and i still cant

here's the line 298:

**Dynamic FlxG.cameras.setDefaultDrawTarget = [camGame];**

if anyone can help me, please comment here

**Comment by ghost** [comment]
What haxe version do you have intalled? I recommend the latest (4.2.1) from the website.
4.1.5 works also but I recommend the latest

**Comment by AshuraTheEchidna** [comment]
> What haxe version do you have intalled? I recommend the latest (4.2.1) from the website.
> 4.1.5 works also but I recommend the latest

im using haxe 4.1.5, i will try to use 4.2.1, ill tell you if it works or not

**Comment by isakube** [comment]
check the lines above that. normally in VSC, 'missing ;' errors don't show up on the line where the ; is actually missing

**Comment by azabiphetamine** [comment]
why was this abandoned

**Comment by TheRealWhitty** [comment]
> 
> 
> What haxe version do you have intalled? I recommend the latest (4.2.1) from the website.
> 4.1.5 works also but I recommend the latest

baka the newest haxe isnt compatible with fnf
